selfdrive/controls/lib/latcontrol_pid.py

```diff
@@ -6,11 +6,12 @@
 
 
 class LatControlPID():
-  def __init__(self, CP):
+  def __init__(self, CP, CI):
     self.pid = PIController((CP.lateralTuning.pid.kpBP, CP.lateralTuning.pid.kpV),
                             (CP.lateralTuning.pid.kiBP, CP.lateralTuning.pid.kiV),
                             k_f=CP.lateralTuning.pid.kf, pos_limit=1.0, neg_limit=-1.0,
                             sat_limit=CP.steerLimitTimer)
+    self.feedforward = CI.get_steer_feedforward
```

Comment:
Is this more efficient? Or should we pass CI to update() and call get_steer_feedforward() there?

Reply:
```suggestion
    self.get_steer_feedforward = CI.get_steer_feedforward
```

Think this is a better name to indicate it's a function not a variable.

Reply:
Think this is fine, no need to pass to update.